src/subcommands/utils.go

```diff
@@ -67,6 +67,29 @@ func writeIndex(indexFilePath string, indexMap map[string]*pb.IndexListEntry) er
 	return nil
 }
 
+func createSearchDictionary(ile *pb.IndexListEntry) error {
```

Comment:
Put this in lexical_search.go, not here. Try not to put anything in here unless it truly doesn't belong elsewhere

Reply:
done